kong/datadog_checks/kong/metrics.py

```diff
@@ -5,17 +5,26 @@
 # https://docs.konghq.com/hub/kong-inc/prometheus/#available-metrics
 METRIC_MAP = {
     'kong_bandwidth': 'bandwidth',
+    'kong_bandwidth_bytes': 'bandwidth.bytes',
     'kong_datastore_reachable': {
         'name': 'datastore.reachable',
         'type': 'service_check',
         'status_map': {'0': 'CRITICAL', '1': 'OK'},
     },
     'kong_http_consumer_status': 'http.consumer.status',
+    'kong_http_requests_total': 'http.requests.total',
     'kong_http_status': 'http.status',
+    'kong_kong_latency_ms': 'kong.latency.ms',
     'kong_latency': 'latency',
     'kong_memory_lua_shared_dict_bytes': 'memory.lua.shared_dict.bytes',
     'kong_memory_lua_shared_dict_total_bytes': 'memory.lua.shared_dict.total_bytes',
+    'kong_nginx_connections_total': 'nginx.connections.total',
+    'kong_nginx_hconnections_total': 'nginx.hconnections.total',
```

Comment:
Openmetrics/prometheus metrics ending with `_total` need to be collected without the `_total` suffix in OMv2, which is what the agent uses by default for the kong integration. 

2nd note here:
https://docs.datadoghq.com/integrations/openmetrics/#configuration

```suggestion
    'kong_http_requests': 'http.requests',
    'kong_http_status': 'http.status',
    'kong_kong_latency_ms': 'kong.latency.ms',
    'kong_latency': 'latency',
    'kong_memory_lua_shared_dict_bytes': 'memory.lua.shared_dict.bytes',
    'kong_memory_lua_shared_dict_total_bytes': 'memory.lua.shared_dict.total_bytes',
    'kong_nginx_connections': 'nginx.connections',
    'kong_nginx_hconnections': 'nginx.hconnections',
```